syntax = "proto3";
package streetgangpb;


message CreateWorldRequest
{
	string track_id = 1;
	string world_name = 2;
}
